app/src/main/java/com/sdp13epfl2021/projmag/database/impl/cache/OfflineCachedCandidatureDatabase.kt

```diff
@@ -113,16 +114,31 @@ class OfflineCachedCandidatureDatabase(
     }
 
     override fun pushCandidature(
-        candidature: Candidature,
+        projectId: ProjectId,
+        userId: String,
         newState: Candidature.State,
         onSuccess: () -> Unit,
         onFailure: (Exception) -> Unit
     ) {
-        val projectId: ProjectId = candidature.projectId
-        val oldList: List<Candidature> = candidatures[projectId] ?: emptyList()
-        candidatures[projectId] = oldList + candidature
+//        val projectId: ProjectId = candidature.projectId
+//        val oldList: List<Candidature> = candidatures[projectId] ?: emptyList()
+//        candidatures[projectId] = oldList + candidature
+//        saveCandidature(projectId)
```

Comment:
These comments can be removed?

Reply:
i've left these as this method still has to be implemented similarly to this